tests/serverpod_test_server/test/defaults/uuid/uuid_default_test.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+import 'package:test/test.dart';
+
+void main() async {
+  group(
+    "Given a class with 'default' UUID fields",
+    () {
+      test(
+        'when an object of the class is created, then the "default=random" UUID field should not be null',
+        () {
+          var object = UuidDefault();
+          expect(object.uuidDefaultRandom, isNotNull);
+        },
+      );
```

Comment:
Suggestion: Add tests for the field `uuidDefaultRandomNull`.

This file does not contain any test for uuidDefaultRandomNull. Is that intentional?

Reply:
This is the same for all tests in this directory.

Reply:
Nope is not intentional, i just missed this field, thank you!